commands.js

```diff
@@ -71,6 +71,10 @@ function Commander (view, cabal) {
           self.view.writeLine.bind(self.view)(`/${key}`)
           self.view.writeLine.bind(self.view)(`  ${self.commands[key].help()}`)
         }
+        self.view.writeLine.bind(self.view)(`ctrl+w`)
```

Comment:
ctrl+j ?

Reply:
thanks! updated to `alt-n` (which is the new one lol)